kalite/settings/base.py

```diff
@@ -248,9 +255,18 @@
         "OPTIONS": {
             "timeout": 60,
         },
+    },
+    "topic_tools": {
+        "ENGINE": "django.db.backends.sqlite3",
+        "NAME": TT_DATABASE_PATH,
+        "OPTIONS": {
+            "timeout": 60,
```

Comment:
Curious... what's this? Could it go with a comment? I see that it's also in the above database.